src/content/extensions/wpgraphql-meta-query.mdx

```diff
@@ -10,6 +10,40 @@ by meta fields.
 
 ## Examples
 
+```
+add_action( 'graphql_register_types', function() {
```

Comment:
ah, shoot. Didn't notice the file this was added to. This page is actually intended to document an extension plugin `https://github.com/wp-graphql/wp-graphql-meta-query`.

We should probably put this example here: https://docs.wpgraphql.com/getting-started/custom-fields-and-meta
🤔

Reply:
@paulisloud ^ mentioning you to make sure you get notified